appveyor.yml

```diff
@@ -17,7 +17,11 @@ build_script:
     if ($LastExitCode -ne 0) { exit $LastExitCode }
 test_script:
 - ps: >-
-    dotnet test test/ProgressOnderwijsUtilsTests/ProgressOnderwijsUtilsTests.csproj -c Release
+    dotnet test test/ProgressOnderwijsUtils.Test -c Release
+
+    if ($LastExitCode -ne 0) { exit $LastExitCode }

```

Comment:
Als je dit met de defaults doet (batch dus) en niet met `- ps: >-` dan schijnt de appveyor config dit bij default al te doen.

Ik ben trouwens voor om hiervan een batch file te maken ivm reproduceerbaarheid buiten appveyor, vooral nu het net niet geheel triviaal meer is.

Reply:
> Als je dit met de defaults doet (batch dus) en niet met - ps: >- dan schijnt de appveyor config dit bij default al te doen.

Waarom denk je dat?

Reply:
https://www.appveyor.com/docs/build-configuration/#script-blocks-in-build-configuration